openslides_backend/action/actions/committee/import_meeting.py

```diff
@@ -0,0 +1,241 @@
+import time
+from typing import Any, Dict, Iterable, Optional, Tuple
+
+from ....models.base import model_registry
+from ....models.fields import (
+    BaseTemplateField,
+    GenericRelationField,
+    GenericRelationListField,
+    RelationField,
+    RelationListField,
+)
+from ....models.models import Committee
+from ....shared.exceptions import ActionException, MissingPermission
+from ....shared.filters import FilterOperator
+from ....shared.interfaces.event import EventType
+from ....shared.interfaces.write_request import WriteRequest
+from ....shared.patterns import KEYSEPARATOR, Collection, FullQualifiedId
+from ...action import Action
+from ...util.crypto import get_random_string
+from ...util.default_schema import DefaultSchema
+from ...util.register import register_action
+from ...util.typing import ActionData, ActionResults
+
+
+@register_action("committee.import_meeting")
+class CommitteeImportMeeting(Action):
+    """
+    Action to import a meeting.
+    """
+
+    model = Committee()
+    schema = DefaultSchema(Committee()).get_update_schema(
+        additional_required_fields={"meeting_json": {"type": "object"}}
+    )
+
+    def perform(
```

Comment:
You do not have to follow the "normal" perform methods here, you can just leave out the parts you don't need. See comments below

Reply:
Removed some parts from the perform method.